tests/test_ops/test_rans.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import pickle
+import time
+from multiprocessing import cpu_count
+
+import pytest
+import torch
+
+from mmcv.ops import (pmf_to_quantized_cdf, rans_decode_with_indexes,
+                      rans_encode_with_indexes)
+
+EXTEND_TIMES = 100
+DEVICES = ['cpu', 'cuda'] if torch.cuda.is_available() else ['cpu']
+NUM_THREDS = [i for i in [1, 2, 4] if i <= cpu_count()]
```

Comment:
https://github.com/open-mmlab/mmcv/blob/3269278ed0ef2f340bb8d5fc3334e85c08139e1d/setup.py#L187

We can call `cpu_affinity` of `psutil` to get the number of CPUs.

See https://docs.python.org/3/library/multiprocessing.html#multiprocessing.cpu_count and https://psutil.readthedocs.io/en/latest/#psutil.Process.cpu_affinity for their difference.